public/scripts/presetScript.js

```diff
@@ -0,0 +1,99 @@
+const presetBox = document.querySelector("#preset-box");
+
+async function savePreset() {
+    // TODO: Add game theme
```

Comment:
Hadden we niet afgesproken dat er geen TODO's in een PR mogen staan?

Reply:
Dat is onderdeel van Luke's oude Preset PR. zoals eerder vermeld: negeer het presets gedeelte.

Reply:
Die TODO is in zijn eigen branch als het goed is ook al verwijderd.

Reply:
correct.